django/webscraper/views.py

```diff
@@ -0,0 +1,25 @@
+"""This module defines views for webscraper application"""
+from django.shortcuts import render, redirect
+
+from .web_scraper import start_scrapping, kill_scrapping
+
+
+def refresh_db_view(request):
+    """View to confirm that one wishes to start scrapping"""
+    if request.method == "POST" and request.POST.get("id", None) == "refresh":
+        start_scrapping()
+        return redirect("/")
```

Comment:
Redirect should point to url name, not an absolute url.

Reply:
Could you give an example?

Reply:
return redirect("index")